frontends/web/src/routes/device/bitbox02/backups.tsx

```diff
@@ -72,18 +72,16 @@ class Backups extends Component<Props, State> {
     if (this.props.backupOnBeforeRestore) {
       this.props.backupOnBeforeRestore(backup);
     }
-    apiPost(
-      'devices/bitbox02/' + this.props.deviceID + '/backups/restore',
-      this.state.selectedBackup).then(({ success }) => {
-      this.setState({
-        restoring: false,
-        errorText: success ? '' : this.props.t('backup.restore.error.general'),
+    restoreBackup(this.props.deviceID, this.state.selectedBackup)
+      .then(({ success }) => {
+        this.setState({
+          restoring: false,
+          errorText: success ? '' : this.props.t('backup.restore.error.general'),
```

Comment:
It would be nice to rebase on https://github.com/digitalbitbox/bitbox-wallet-app/pull/1933/files after approval and maybe use the error message here instead of the generic one. But it is not localized, so I am not sure about the last part. wdyt?
cc @benma

Reply:
Ok, I'll merge this PR as is, and then we do the nice error message after #1933